pkgs/by-name/op/openbao/package.nix

```diff
@@ -0,0 +1,83 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  installShellFiles,
+  nixosTests,
+  makeWrapper,
+  gawk,
+  glibc,
+  testers,
+  openbao,
+}:
+
+buildGoModule rec {
+  pname = "openbao";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "openbao";
+    repo = "openbao";
+    rev = "v${version}";
+    hash = "sha256-2dylOra/XuAQMIeb05Lp2nqLxXhPmSK+4poSaZ0GzJs=";
+  };
+
+  vendorHash = "sha256-vdjoYw0VWNdMYJERFj2RAO63WbDShgeyNili5BFEPUk=";
+
+  proxyVendor = true;
+
+  subPackages = [ "." ];
+
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+  ];
+
+  tags = [
+    "openbao"
+    "bao"
+  ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/openbao/openbao/version.GitCommit=${src.rev}"
```

Comment:
As a note, I think you're short two more flags:

- `-X github.com/openbao/openbao/version.fullVersion=...`
- `-X github.com/openbao/openbao/version.BuildDate=...`

You could probably compute build date from the commit date if you want ([we use the build's actual date upstream](https://github.com/openbao/openbao/blob/main/.goreleaser-template.yaml#L29)).

hth!

Reply:
We recently did some testing of this at GitLab and found BuildDate was completely ignored so I think you're fine leaving it off. But I think fullVersion is definitely necessary.